java-logging-httpcomponents/src/test/java/uk/gov/hmcts/reform/logging/httpcomponents/OutboundRequestLoggingInterceptorTest.java

```diff
@@ -26,20 +28,20 @@
 
     private static final ProtocolVersion ANY_PROTOCOL = new ProtocolVersion("any", 0, 0);
 
-    private TestAppender testAppender = new TestAppender();
+    private final TestAppender testAppender = new TestAppender();
 
     @Before
-    public void addAppender() throws Exception {
+    public void addAppender() {
         ((Logger) LoggerFactory.getLogger(ROOT_LOGGER_NAME)).addAppender(testAppender);
     }
 
     @After
-    public void removeAppender() throws Exception {
+    public void removeAppender() {
         ((Logger) LoggerFactory.getLogger(ROOT_LOGGER_NAME)).detachAppender(testAppender);
     }
 
     @Test
-    public void logsRequestAndResponseFields() throws Exception {
+    public void logsRequestAndResponseFields() throws IOException, HttpException {
```

Comment:
which tool complained about this?

Reply:
PMD: should not explicitly throw/catch raw Exceptions

Reply:
I think it would be good to add PMD plugin and ruleset to this repo then (for local checks).

See https://github.com/hmcts/spring-boot-template for reference.

Reply:
it would. separate pr for sure